src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -193,12 +191,14 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
         validateFagsystem(ytelsestype, fagsystem)
         val eksternFagsakId = opprettTilbakekrevingRequest.eksternFagsakId
         val eksternId = opprettTilbakekrevingRequest.eksternId
+        val erManueltOpprettet = opprettTilbakekrevingRequest.manueltOpprettet
```

Comment:
vil denne nokon gong være true? dersom førstegongsbehandlinger no skal opprettast async, så kan vel denne eigentleg fjernast frå OpprettTilbakekrevingRequest?

Reply:
Fagsystem sender alltid true, men denne samme kontrakten blir brukt av familie-tilbake når behandling oppretter manuelt/batch. Denne koden blir gjenbrukt for alle 3 måter. Så det kan være false, når familie-tilbake kaller den.